alerts/forward.libsonnet

```diff
@@ -51,6 +51,32 @@
               message: 'CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of forward requests to {{ $labels.to }}.',
             },
           },
+          {
+            alert: 'CoreDNSForwardHealthcheckFailureCount',
+            expr: |||
+              rate(coredns_forward_healthcheck_failure_count_total[10s]) > 3
```

Comment:
We should add a `%(corednsSelector)s` selector here, like all the above alerts have. Also lets make this be 5 minute range selector instead of 10 seconds.

Reply:
Commit `5cbc692` addresses your comment here.